bot_components/slack_comm.py

```diff
@@ -8,7 +8,7 @@
 """
 import json
 
-import bot_components
+from .. import bot_components
```

Comment:
@bboe : What do you suggest as a solution to this problem? 

I'm running the testing inside of the docker image.